bspump/elasticsearch/sink.py

```diff
@@ -50,10 +82,40 @@ def process(self, context, event):
 			context.get("es_index", self.Index),
 			event.pop("_id", None),
 			orjson.dumps(event, option=orjson.OPT_APPEND_NEWLINE),
-			bulk_class=self.BulkClass
+			data_feeder=self.DataFeeder,
+			bulk_class=self.BulkClass,
 		)
 
 
+	async def _data_feeder_create(self, items):
```

Comment:
where feeders shouldn't be part of the class ... there is no interaction with `self` as far a I can tell.